varnish/varnish.spec

```diff
@@ -258,7 +285,7 @@ rm -rf %{buildroot}
 %doc doc/changes*.html
 
 %pre
-getent group varnish    >/dev/null || groupadd -r varnish
```

Comment:
Please add `%{service_user}` and `%{service_group}` variables to define service user/group.